.github/workflows/zips.yml

```diff
@@ -113,59 +116,98 @@ jobs:
+      - name: Check the comment exists
+        id: find-comment
+        uses: peter-evans/find-comment@v2
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: github-actions[bot]
+          body-includes: Build files for
+      - name: Get comment body
+        id: get-comment-body
+        # Setting a multi-line output requires escaping line-feeds. See <https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings>.
         run: |
-          git add .
-          git status
-          git diff --staged --quiet && echo 'No changes to commit; exiting!' && exit 0
-          git pull --no-edit --quiet
-          git commit -m "Build and publish ${{ github.ref }}"
-          git push origin master
+          body="Build files for ${{ github.event.pull_request.head.sha }} are ready:
+          - Download [development build](https://storage.googleapis.com/${{ env.BUCKET_NAME }}/builds/${TARGET_DIR}/google-site-kit-dev.zip?${{ github.sha }})
+          - Download [production build](https://storage.googleapis.com/${{ env.BUCKET_NAME }}/builds/${TARGET_DIR}/google-site-kit.zip?${{ github.sha }})"
+          delimiter="${body//$'\n'/'%0A'}"
+          echo "body<<${delimiter}" >> $GITHUB_OUTPUT
+          echo "$body" >> $GITHUB_OUTPUT
+          echo "${delimiter}" >> $GITHUB_OUTPUT
         env:
-          GIT_AUTHOR_EMAIL: ${{ github.actor }}@users.noreply.github.com
-          GIT_AUTHOR_NAME: ${{ github.actor }}
-          GIT_COMMITTER_EMAIL: ${{ github.actor }}@users.noreply.github.com
-          GIT_COMMITTER_NAME: ${{ github.actor }}
+          TARGET_DIR: ${{ steps.branch-target.outputs.dir }}${{ steps.pr-target.outputs.dir }}
+      - name: Create comment on PR with links to build files
+        if: ${{ steps.find-comment.outputs.comment-id == '' }}
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body: ${{ steps.get-comment-body.outputs.body }}
+      - name: Update comment on PR with links to build files
+        if: ${{ steps.find-comment.outputs.comment-id != '' }}
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          comment-id: ${{ steps.find-comment.outputs.comment-id }}
+          edit-mode: replace
+          body: ${{ steps.get-comment-body.outputs.body }}
 
-  remove-pr:
+  remove-zips:
     name: Remove Zips
     runs-on: ubuntu-latest
     timeout-minutes: 20
     if: ( github.event_name == 'pull_request' && github.event.action == 'closed' && contains( github.head_ref, 'dependabot/' ) == false ) && github.event.pull_request.head.repo.fork == false
     steps:
-      - uses: actions/checkout@v3
+      - name: Set branch environment variables
+        id: branch-target
+        if: github.event_name != 'pull_request'
+        run: echo "dir=${GITHUB_REF##*/}" >> $GITHUB_OUTPUT
```

Comment:
We probably don't need to automate the cleanup of builds for branches here, as I'm not sure this workflow even runs for those events, but we also rarely use them so it's less important. Also the tester doesn't currently expose branches other than main/develop.

Reply:
Good point. Updated.